projects/arlas-toolkit/src/lib/components/reconnect-dialog/reconnect-dialog.component.html

```diff
@@ -1,10 +1,10 @@
 <div *ngIf="code === 401" class="code-401">
     <h2 mat-dialog-title>{{'Session Expired' | translate}}</h2>
-    <mat-dialog-content>{{'You are no more connected or your session has expired. Would you like to log in again?'| translate}}</mat-dialog-content>
+    <mat-dialog-content>{{'You are no longer logged in or your session has expired. Would you like to log in again?'| translate}}</mat-dialog-content>
```

Comment:
We can only say : "You are no longer logged in. Would you like to log in again?"
There is no need to add session information because, for the user, their current status is disconnected